lib/Models/TerriaJsonCatalogFunction.js

```diff
@@ -185,13 +186,54 @@ function TerriaJsonCatalogFunction(terria) {
    */
   this.url = undefined;
 
+  /*
+   * Gets or sets the part of a HTTP 202 response to poll again for content.
+   * A HTTP 202 response is returned as an object with 
+   * {
+   *   status: 202
+   *   headers: responseHeaders
+   *   respone: responseBody
+   * }
+   * where
+   * responseHeaders is an object with key-value pairs of all HTTP response 
+   * 	headers
+   * responseBody is the unaltered XMLHttpRequest.response
+   * If you were to supply a 202 response with the url as a location header,
+   * acceptedUrl can be specified as "headers.location" to poll it again
+   * after acceptedDelay
+   * If you were to supply a 202 response with the url as a property of a
+   * JSON object in the response body, for example
+   * {
+   *   prop1: {
+   *     prop2: "something",
+   *     prop3: url
+   *   }
+   * }
+   * acceptedUrl can be specified as response.prop1.prop3
+   * If you were to supply a 202 response with the url as plain text in the 
+   * response body, acceptedUrl can be specified as response (i think)
+   * If you were to supply a 202 response with the url as a property of some 
+   * other mime type/format, eg. XML, YAML
+   * This is open source software, add the code yourself.
+   * @type {String}
+   */
+  this.acceptedUrl = undefined;
+
+  /*
+   * Gets or sets the time in milliseconds to wait before polling a 
+   * HTTP 202 response acceptedUrl
+   * default 10 seconds
+   * @type {Number}
+   */
+  this.acceptedDelay = 10000;
+
   /**
    * Gets or sets the input parameters to the service.
    * @type {FunctionParameter[]}
    */
   this.inputs = [];
 
-  knockout.track(this, ["url", "inputs"]);
+  knockout.track(this, ["url", "acceptedUrl", "aceptedDelay", "inputs"]);
```

Comment:
As far as I can tell, these don't need to be tracked as observables by knockout - nothing needs to react to them changing.

Reply:
I thought that was how to get them from the init file. What is another way?